BinaryEncoding.md

```diff
@@ -584,6 +584,8 @@ natural alignment. The bits after the
 are reserved for [future :unicorn:][future threads] use
 (e.g., for shared memory ordering requirements).
 
+It is a validation error if `offset + numBytesTouched(load/store) - 1 > 2^32 - 1`. If this condition is true, the load or store is guaranteed to be out of bounds.
```

Comment:
How about `sizeof(memoryAccess)` instead? We use `sizeof` in this page. I'd also only talk about "memory access" instead of "load or store" so that we don't have to change the text when we add atomics.